tss-esapi-sys/build.rs

```diff
@@ -9,11 +9,21 @@ fn main() {
 
     cfg_if::cfg_if! {
         if #[cfg(feature = "generate-bindings")] {
+            #[cfg(feature = "bundled")]
+            let installation = tpm2_tss::Installation::bundled();
```

Comment:
I'm a bit stumped about how the linking and bundling is supposed to be achieved in this branch. For the `else` below I can see that the call to `pkg_config` leads to them being statically linked, but here the `output_linker_arguments` is 1) not "bundled"-specific, and 2) it links dynamically to those libraries. Seems that the work to prepare the libs is done, but the linking isn't? 

Maybe we're just missing another conditional on `cfg(feature = "bundled")` to call `output_linker_arguments` vs `pkg_config`.